services/data-info/src/data_info/services/page_tabular.clj

```diff
@@ -27,12 +27,15 @@
 
 (defn- seek-prev-line
   [chunk start-pos]
+  (log/debug "seeking for a line-ending backwards starting at" start-pos)
```

Comment:
The changes in this file are primarily just adding logging lines such as this. There's some lurking but very unlikely to be triggered bugs in our CSV parsing which this may be relevant for in the future.

(specifically: if the chunk size provided is small enough, and/or the line of a tabular data file long enough, that the chunk includes only one linebreak, the endpoint will return the partial line that linebreak ends as though it is a complete line. it's unlikely to come up in practice because our real-world code paths have a minimum chunk size of 8KB)